File: TD/TD1/Code/6.cpp
/**
 * Simuler le fonctionnement d'une calculatrice simple
 */
#include <iostream>

int main()
{
  int a;
  int b;
  char c;

  std::cout << "Donne moi première valeur : " << std::flush;
  std::cin >> a;
  std::cout << "Choisi un operateur parmi : + , - , * et / : " << std::flush;
  std::cin >> c;
  std::cout << "Donne moi deuxième valeur : " << std::flush;
  std::cin >> b;

  switch (c) {
  case '+':
    std::cout << a << " + " << b << " = " << a + b << std::endl;
    break;
  case '-':
    std::cout << a << " - " << b << " = " << a - b << std::endl;
    break;
  case '*':
    std::cout << a << " * " << b << " = " << a * b << std::endl;
    break;
  case '/':
    if (b == 0)
      std::cout << "Operation impossible !" << std::endl;
    else
      std::cout << a << " / " << b << " = " << a / b << std::endl;
    break;
  default:
    std::cout << "Unknown error !" << std::endl;;
    break;
  }

  return 0;
}

File: TD/TD1/Code/Python/6.py
#!/usr/bin/python
# Simuler le fonctionnement d'une calculatrice simple
print('Donne moi une première valeur : ', end='')
a = int(input())
print('Donne moi une deuxième valeur : ', end='')
b = int(input())
print('Choisie une opération : + , -, *, / : ', end='')
c = str(input())

if c == '+':
    print(a, '+', b, '=', a + b)
elif c == '-':
    print(a, '-', b, '=', a - b)
elif c == '*':
    print(a, '*', b, '=', a * b)
elif c == '/':
    #  Division par zéro
    if b == 0:
        print('Opération impossible')
    else:
        print(a, '/', b, '=', a / b)
else:
    print('Une erreur inconnue est survenue')
